### DISCUSSION TITLE
New Times Series doesn't show all values in tooltip popup

### DISCUSSION THREAD

**Discussion by winnieXY** [grafana/grafana#39939] [Dashboards, Panels & Visualizations]
**What happened**:
I'm using grafana for house automation things and some of my sensors ony submit data every ~1h. With the old Graph the last non-null value of all values were shown in the tooltip - within the new timeseries only some of the values are shown which are in close proximity to the cursor.

**What you expected to happen**:
That there is an option to optionally enable the old behaviour and display the last non-null value in the tooltip for all values. I'm aware that some users consider the old behaviour to be faulty. I'm also aware of the following bug reports #36346 and #34552

**How to reproduce it (as minimally and precisely as possible)**:
Switch for a given panel from graph to time-series

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.1.4 (proximity increased from 10 to 15 - I'm aware of the PR)
- Data source type & version: influxdb 1.8.9
- OS Grafana is installed on: debian buster
- User OS & Browser: div. 
- Grafana plugins: time series / graph
- Others:

**Comment by thomasvnl** [comment]
I too am missing this behaviour. I mean, the legend can show the last non-null/numeric value (Last*), so why can't the tooltip? Its not like the data is not there (assuming that its within the time range). 

This could then also be complemented with a similar feature that is also available for connecting null values in series that have gaps in them that are limited to within a certain amount of time. Anything above that time threshold, no last value is shown in the tooltip and thus there could be a fault with the data of a sensor not broadcasting.

It seems like Grafana is already addressing series that have intermittent data i.e. data that is not sent on regular periods in such a manner. Therefor I vote to bring back the behaviour to the time series graphing tooltip as well.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
